src/main/java/org/embulk/output/elasticsearch/ElasticsearchHttpClient.java

```diff
@@ -289,6 +305,34 @@ private void deleteIndex(String indexName, PluginTask task)
         }
     }
 
+    private void deleteAlias(String indexName, String aliasName, PluginTask task)
```

Comment:
Unused method?

Reply:
I used it in this test class method https://github.com/joe-td/embulk-output-elasticsearch/blob/10d23386935df5f0d99ac94aa3b4836b78753062/src/test/java/org/embulk/output/elasticsearch/ElasticsearchTestUtils.java#L75